lib/taurus/qt/qtgui/__init__.py

```diff
@@ -54,22 +54,45 @@
 # Note: this is an experimental feature introduced in v 4.3.0a
 # It may be removed or changed in future releases
 
+class LazyModule(ModuleType):
+
+    def __init__(self, name, package, entry_point):
+        super(LazyModule, self).__init__(name)
+        self.__package__ = package
```

Comment:
Do we really need to overwrite the  `__package__` special attribute of `LazyModule`? 

Unless for some reason we want to overwrite, I think it is safer not to, and just store our "target" package name in `self._target_pkg_name` or something similar.

Reply:
I added it based on information [here](https://docs.python.org/3.6/library/types.html#types.ModuleType). However, this may not be needed as I did not tested it without it

Reply:
Oh. I was not aware of this. Thanks for pointing to the docs. I agree with your implementation.